app/views/tracking/OffPage.js

```diff
@@ -0,0 +1,69 @@
+import { useNavigation } from '@react-navigation/native';
+import React from 'react';
+import {
+  Dimensions,
+  ImageBackground,
+  StatusBar,
+  Text,
+  View,
+} from 'react-native';
+import { SvgXml } from 'react-native-svg';
+
+import BackgroundImage from './../../assets/images/launchScreenBackground.png';
+import StateUnknown from './../../assets/svgs/stateUnknown';
+import { Typography } from './../../components/Typography';
+import Colors from './../../constants/colors';
+import ButtonWrapper from '../../components/ButtonWrapper';
+import languages from '../../locales/languages';
+import MayoInfo from './MayoInfo';
+import styles from './style';
+
+const buttonLabel = languages.t('label.home_enable_location');
+const size = Dimensions.get('window').height;
+
+export default function OffPage() {
```

Comment:
no default exports and use the `export const OffPage = () => {}` format to be consistent with other files.

I'll eventually write this up as a wiki style guide!